### PR TITLE
Fixed Scheduler URI

### PR DESCRIPTION
**Pull request by crackcode123**
Remove SCHEDULER_URI from Cassandra svc.yml
https://jira.mesosphere.com/browse/INFINITY-3146

### CODE REVIEW

File: frameworks/cassandra/universe/marathon.json.mustache [hunk 1]
```diff
@@ -154,7 +154,6 @@
     "TASKCFG_ALL_REMOTE_SEEDS": "{{service.remote_seeds}}",
     "EXECUTOR_URI": "{{resource.assets.uris.executor-zip}}",
     "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
-    "SCHEDULER_URI": "{{resource.assets.uris.scheduler-zip}}",
     "CASSANDRA_URI": "{{resource.assets.uris.cassandra-tar-gz}}",
     "LIBMESOS_URI": "{{resource.assets.uris.libmesos-bundle-tar-gz}}",
```
Comment (nickbp) [inline frameworks/cassandra/universe/marathon.json.mustache]: While in here, can also remove this LIBMESOS_URI envvar as it shouldn't be passed to tasks either (only used by scheduler)

**Comment by crackcode123** [comment]
Fixed LIBMESOS_URI

**Comment by nickbp** [comment]
It would make sense to also backport this fix to the `sdk-0.40` branch

**Comment by crackcode123** [comment]
Nick please merge it - all test are passed.

**Comment by nickbp** [comment]
FWIW we haven't been too particular about who hits the green button when something is green+approved

**Pull request merged**